### ISSUE TITLE
Seedvault cannot restore apps on divested-20.0-20231110-dos-crosshatch

### ISSUE THREAD

**Issue by wreps8Owt** [Divested-Mobile/DivestOS-Build#271]
Today, restoring a seedvault backup with apps included failed on divested-20.0-20231110-dos-crosshatch, in a manner similar to https://github.com/seedvault-app/seedvault/issues/588 .

However, restoring after downgrading to divested-20.0-20230907-dos-crosshatch via fastboot works well.

**Comment by SkewedZeppelin** [comment]
Yep, I've had entire restores fail myself because of one app, it is annoying how unreliable it is.

Best to use SV as an extra in hope it works, and fall back to manual restore: https://divestos.org/pages/faq#backups
